src/main/java/com/puppycrawl/tools/checkstyle/utils/CommonUtils.java

```diff
@@ -467,4 +467,31 @@ public static String getFileExtension(String fileNameWithExtension) {
         }
         return extension;
     }
+
+    /**
+     * Checks whether the given string is a valid identifier.
+     * @param str A string to check.
+     * @return true when the given string contains valid identifier.
+     */
+    public static boolean isIdentifier(String str) {
+        boolean isIdentifier = !str.isEmpty() && Character.isJavaIdentifierStart(str.charAt(0));
```

Comment:
@rnveach Have you noted the difference between the 'isJavaIdentifierStart', which is used for the first symbol only and the 'isJavaIdentifierPart', which is used for all other symbols? If so, I already present the example of the loop in the first reply to your comment. Is it ok?